src/utils/index.ts

```diff
@@ -56,9 +62,8 @@ export function formatCode(
 ) {
   return code.replace(/{{(.*?)}}/g, (match) => {
     const name = match.split(/{{|}}/).filter(Boolean)[0];
-    return JSON.stringify(
-      args?.[name] !== undefined ? args?.[name] : optionalParams?.[name],
-    );
+
+    return JSON.stringify(args?.[name] ?? optionalParams?.[name] ?? undefined);
```

Comment:
What's the intention of this change? Or is this just a syntax change that achieves the same behavior?

Reply:
it achieves the same behavior, but we don't need to filter inputs anymore, how we did it previously.
It's also cleaner to read in my opinion:
is it passed in ? otherwise is there an optional param that's not null'isch ? otherwise return undefined.